### ISSUE TITLE
Theme entirely set to black

### ISSUE THREAD

**Issue by MewMero** [Retrospring/retrospring#1462]
### Steps to reproduce the problem

I was originally changing the theme to how I would like it.

### Expected behaviour

The colors of everything changed within the themes tab should had changed everything on the website through my account.

### Actual behaviour

Everything turned black.

### Specifications

Version 1.60.110 Chromium: 119.0.6045.105 (Official Build)

**Comment by pixeldesu** [comment]
We hear this happening very seldomly, but have not been able to reproduce this ourselves yet. Did the colors only turn black after saving?

**Comment by MewMero** [comment]
Yes, I retry it multiple times even a day after. I would change most of the parts within the theme tab and after saving, everything would be pitch black.

**Comment by pixeldesu** [comment]
Hmm, would you mind sharing what colors you set it to? (either by text, or a screenshot of the values/theme page before saving)

Not sure if that will help, but maybe it helps resolving the problem so that it doesn't happen anymore. Any pointer to what the issue could be is going to help in this case.

Also, if you have trouble using the site because of it being all-black, you can tell me your username and I can reset your theme to the default one.

**Comment by MewMero** [comment]
<img width="814" alt="Screen Shot 2023-11-12 at 3 36 47 PM" src="https://github.com/Retrospring/retrospring/assets/136536309/5b4e4f97-a101-4017-a1ef-d1b19c2bc1e4">
<img width="826" alt="Screen Shot 2023-11-12 at 3 36 38 PM" src="https://github.com/Retrospring/retrospring/assets/136536309/da5a3b84-8fe4-4b48-9811-117de7ed2a35">
<img width="861" alt="Screen Shot 2023-11-12 at 3 36 28 PM" src="https://github.com/Retrospring/retrospring/assets/136536309/2fa54e9d-ef93-4946-8aa7-d6a487da18ff">
<img width="826" alt="Screen Shot 2023-11-12 at 3 36 17 PM" src="https://github.com/Retrospring/retrospring/assets/136536309/7e48548e-9f44-442d-839d-cdd93e5ecf8a">
